loopchain/channel/channel_inner_service.py

```diff
@@ -429,12 +434,17 @@ def crash_callback_in_join_thread(process: ModuleProcess):
         logging.info(f"Channel({ChannelProperty().name}) TX Receiver: initialized")
 
     def update_sub_services_properties(self, **properties):
-        logging.info(f"properties {properties}")
-        stub = StubCollection().channel_tx_creator_stubs[ChannelProperty().name]
-        asyncio.run_coroutine_threadsafe(stub.async_task().update_properties(properties), self.__loop_for_sub_services)
+        logging.debug(f"properties {properties}")
```

Comment:
Is there any reason I need to handle both logic in the update_sub_services_properties function?

I think there is a problem with the checklogic below.
`if len(properties) == 1 and "reject_create_tx" in properties:
            target_stubs.pop('tx_receiver')`